cce/go.mod

```diff
@@ -0,0 +1,8 @@
+module enriquecedores
```

Comment:
Acho que você não precisa de um go.mod aqui, ele pode ficar na raiz. Isso vai simplificar muito a utilização do código a medida que você for adicionando mais pacotes.